ExRouter.js

```diff
@@ -40,7 +41,7 @@ export class ExRouteAdapter {
             this.renderRightButton = this.route.props.renderRightButton.bind(this.route);
         }
         if (this.route.props.renderTitle){
-            this.renderTitle = this.route.props.renderTitle.bind(this.route);
+            this.renderTitle = this.route.props.renderTitle.bind(this, this.route);
```

Comment:
If route should be passed as the first argument to renderTitle then this update should be correct.